beets/mediafile.py

```diff
@@ -1643,6 +1643,12 @@ def update(self, dict):
         StorageStyle('COMPOSERSORT'),
         ASFStorageStyle('WM/Composersortorder'),
     )
+    work = MediaField(
+        MP3DescStorageStyle(u'Work'),
+        MP4StorageStyle('----:com.apple.iTunes:Work'),
+        StorageStyle('WORK'),
+        ASFStorageStyle('beets/Work'),
+    )
```

Comment:
Just to check: does Picard have a tag mapping for the work name? If so, we should do our best to match.

Reply:
It looks like the iTunes one will want changing to `com.apple.iTunes:MusicBrainz Work Id`, and the "WORK" one will want changing to `MUSICBRAINZ_WORKID`, as these are the names [used by Picard](https://github.com/metabrainz/picard/blob/4983631f700c65786ea61b9e19c8a98a20205bb0/picard/formats/mp4.py#L95).

Reply:
If I remember well I got the tag mapping from picard, but upon checking it turns out picard now has a tag for work title (Picard >=1.3) and MusicBrainz Work Id and a tag called Show Work & Movement (Picard>=2.1, only populated by some plugins). I'm updating, could you please check? I'm not 100% sure if it is correct.

Reply:
you find picards tag mapping here: https://picard.musicbrainz.org/docs/mappings/

Reply:
There is a tag conflict between Work and Grouping: in picard, if you take the option "Save iTunes compatible grouping and work" (since Picard>=2.1.0) then the ID3v2 tag for grouping becomes GRP1 and the tag for work TIT1, if it isn't enabled then the tags are TIT1 and  TXXX:WORK  respectively. I chose the first one, but you might disagree. I feel like if we have to choose, let's be ITunes compatible.

Reply:
This conflict and the subsequent change I made to the grouping tag seems to be the cause of the tests failing, an idea on how I could correct it? One would be to keep the old tags but I'm not really satisfied by this, do you have an opinion?

Reply:
Note that we already ported this particular change to MediaFile here: beetbox/mediafile#8